ndcube/tests/test_ndcube.py

```diff
@@ -207,23 +211,27 @@
       ('custom:pos.helioprojective.lat', 'em.wl'),
       {'time': {'axis': None, 'value': u.Quantity(1, unit=u.pix)},
        'hello': {'axis': 0, 'value': u.Quantity(range(int(cubem.dimensions[1].value)), unit=u.pix)},
-       'bye': {'axis': 1, 'value': u.Quantity(range(int(cubem.dimensions[2].value)), unit=u.pix)}}
+       'bye': {'axis': 1, 'value': u.Quantity(range(int(cubem.dimensions[2].value)), unit=u.pix)},
+       'custom:pos.helioprojective.lon': {'axis': None, 'value': u.Quantity(1., unit=u.deg)}
+      }
       ),
      (cube[:, 1],
       NDCube,
       mask_cube[:, 1],
-      _wcs_slicer(wt, [True, False, False, False], (slice(None, None, None), 1)),
+      _wcs_slicer(wt, [False, False, False, True], (slice(None, None, None), 1)),
```

Comment:
```wt``` only has 3 dimensions.  So I'm pretty sure ```[False, False, False, True]``` should be ```[False, False, False]```.

Also, the [definition of cube's missing_axes](https://github.com/sunpy/ndcube/pull/151/files#diff-278d0480e7f7cfc0504f5a11d64496abR119) should be deleted.